### ISSUE TITLE
[Bug] Notifications won't stop appearing

### ISSUE THREAD

**Issue by abacao** [YTVanced/Vanced#1292]
**Bug description**
Several notifications are re-appearing not only after removed but also even if I won't dismiss them.

**Variant**
Non-Root

**Vanced version**
v16.29.39

**Android version**
Android 10

**Device**
Samsung Galaxy S9+

**Steps to Reproduce**
Steps to reproduce the error:
1. Dismiss all notifications
2. Notifications reappear
3. Even without touching on notifications they reappear all at once.

**Expected behavior**
After dismissed, the notifications for the same notification, shouldn't be appearing again.

**Comment by milindgoel15** [comment]
Duplicate of #1107
